packages/backend/prisma/schema.prisma

```diff
@@ -18,17 +18,19 @@ model Link {
 }
 
 model Collection {
-  id        String @id @default(uuid())
+  id        String   @id @default(uuid())
   links     Link[]
   iconUrl   String
   bannerUrl String
   slug      String
+  createdAt DateTime @default(now())
```

Comment:
Yea, this is needed indeed. Maybe you can discovery how we could outsource this  to put it in every other Model as well. "Don't repeat yourself" principle...

Reply:
Unfortunately prisma now don't support this feature (or i can't find it on the internet and with chatGpt)